drivers/net/telnet.c

```diff
@@ -306,7 +306,6 @@ static void telnet_check_ctrlchar(FAR struct telnet_dev_s *priv,
           signo = SIGINT;
           break;
         }
-      else
 #endif
 #ifdef CONFIG_TTY_SIGSTP
```

Comment:
In this case, the coding standard will require a blank line after the #endif.  nxstyle should report this.

Reply:
I will provide a patch fix this soon.